include/grpcpp/security/tls_credentials_options.h

```diff
@@ -103,7 +103,10 @@ class TlsCredentialReloadArg {
   void set_cb_user_data(void* cb_user_data);
   void set_pem_root_certs(const grpc::string& pem_root_certs);
   void add_pem_key_cert_pair(
-      TlsKeyMaterialsConfig::PemKeyCertPair pem_key_cert_pair);
+      const TlsKeyMaterialsConfig::PemKeyCertPair& pem_key_cert_pair);
```

Comment:
In the above comments - "They modify the fields of the underlying C arg...", it seems that they are are implementation details (i..e, C-core) that should not be exposed to the caller.

Reply:
I removed any reference to the "C arg" from the getters and setters (though I left one by the constructor that takes in a C arg as input).